pennylane/templates/subroutines/select_pauli_rot.py

```diff
@@ -21,19 +21,26 @@
 
 
 class SelectPauliRot(Operation):
-    r"""Applies the multiplexer with Pauli rotations.
+    r"""Applies individual single-qubit Pauli rotations depending on the state of
+    designated control qubits.
 
-    This operator, also called a "multiplexed rotation", applies a sequence of uniformly controlled
-    rotations to a target qubit. The rotations are selected based on the values encoded in the control qubits.
+    This operator, also called a **multiplexed rotation** or **uniformly controlled rotation**,
```

Comment:
Any reasons against adding aliases `qml.Multiplexer` and `qml.UniformlyControlledRotation`? We could benefit from users landing on our docs when they google these operations.

Reply:
`qml.Multiplexer` should be an alias for `qml.Select`.
`qml.UniformlyControlledRotation` or `qml.MultiplexedRotation` could be an alias for `SelectPauliRot` :) 

cc @KetpuntoG @josh146, who will have opinions, I'd anticipate. Also maybe cc @co9olguy ? :)

Reply:
these gates go by so many names that aliases are probably a good call (assuming they are ones we've seen used and we're not just making up new ones)

Reply:
see #7585